src/helix-ui/elements/HXFileTileElement.less

```diff
@@ -161,14 +161,18 @@
   }
 }
 
+// TODO: hook for progress bar fill color
 :host([progress]) {
-  #hxLink {
-    color: @gray-600;
```

Comment:
Moved color definition to Light DOM CSS to allow for css inheritance to take over.